testing/endpoint.go

```diff
@@ -730,7 +730,19 @@ func (endpoint *Endpoint) SetChannelState(state channeltypes.State) error {
 	channel := endpoint.GetChannel()
 
 	channel.State = state
-	endpoint.Chain.App.GetIBCKeeper().ChannelKeeper.SetChannel(endpoint.Chain.GetContext(), endpoint.ChannelConfig.PortID, endpoint.ChannelID, channel)
+	endpoint.SetChannel(channel)
```

Comment:
@DimitrisJim I noticed that we could use the helper function here as well, so I pushed the change in your branch. I hope that's ok!